README.md

```diff
@@ -216,6 +216,10 @@ Some configuration options can be specified individually for each target. You ca
   the environment variable will only be set for direct builds of the target via cmake, and not for any
   build where cargo built the crate in question as a dependency for another target.
   The environment variables may contain generator expressions.
+- `corrosion_add_target_generated_headers(<target_name>)`: Use this for crates whose build scripts generate C/C++
```

Comment:
As far as I understand the current approach only works for Rust libraries (INTERFACE_INCLUDE_DIRECTORIES), right? If yes, then that should also be documented here.